models/spring-ai-openai/src/main/java/org/springframework/ai/openai/api/OpenAiApi.java

```diff
@@ -407,29 +407,44 @@ public record ResponseFormat(
 	/**
 	 * Message comprising the conversation.
 	 *
-	 * @param content The contents of the message.
+	 * @param rawContent The contents of the message. Can be either a {@link MediaContent} or a {@link String}.
+	 * The response message content is always a {@link String}.
 	 * @param role The role of the messages author. Could be one of the {@link Role} types.
 	 * @param name An optional name for the participant. Provides the model information to differentiate between
-	 * participants of the same role.
+	 * participants of the same role. In case of Function calling, the name is the function name that the message is
+	 * responding to.
 	 * @param toolCallId Tool call that this message is responding to. Only applicable for the {@link Role#TOOL} role
 	 * and null otherwise.
 	 * @param toolCalls The tool calls generated by the model, such as function calls. Applicable only for
 	 * {@link Role#ASSISTANT} role and null otherwise.
 	 */
 	@JsonInclude(Include.NON_NULL)
 	public record ChatCompletionMessage(
-			@JsonProperty("content") String content,
+			@JsonProperty("content") Object rawContent,
```

Comment:
I don't see the impact of this change to `Object`.  If this can be `MediaContent`, then where is the code that is surfaces that type?